database/migrations/2021_04_21_102707_change_statuses_to_accept_business_checkin_as_integer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeStatusesToAcceptBusinessCheckinAsInteger extends Migration
+{
+    public function up() {
+        Schema::table('statuses', function(Blueprint $table) {
+            $table->integer('business')
+                  ->default(0)
+                  ->change();
+        });
+
+        DB::table('statuses')
+            ->whereNull('business')
+            ->update(['business' => '0']);
+        DB::table('statuses')
+            ->where('business', 'true')
+            ->update(['business' => '1']);
+        DB::table('statuses')
+            ->where('business', 'false')
+            ->update(['business' => '0']);
```

Comment:
```suggestion
```
We don't have "true" or "false" in our database. There are only the numbers 0 and 1, so in my opinion this migration is unnecessary.

Reply:
It's actually defined as boolean in the first database migration, so I'm unsure about that.

Reply:
A boolean in Laravels models is just an integer in the database.
So it has no affect.

Reply:
These are the current values of our productive database: 
![image](https://user-images.githubusercontent.com/4103693/116410647-44723300-a835-11eb-8140-5d9a275b9116.png)

Reply:
>  So it has no affect.
*effect. But yeah, I guess. I'll change that accordingly.